lib/utils/getInfuraUrl/index.ts

```diff
@@ -1,4 +1,4 @@
-import { urls } from "@klimadao/lib/constants";
+import { urls } from "../../constants";
```

Comment:
`lib` package should not depend on any node env variables. Goal is to keep the `lib` package totally agnostic of the environment (someday, published as a standalone component library to be used by the community)

So we can instead export a util that accepts the infuraId which is passed down at run time from `site` or `app`

The infuraId should be a required param. The two functions can be combined into one

```ts
export const getInfuraUrl = (params: { chain?: "polygon" | "eth", infuraId: string }) => {
  const { chain = "polygon" } = params; // fallback to polygon as default
  const baseUrl = chain === "eth" ? urls.infuraEth : urls.infuraRpc;
  return `${baseUrl}/${params.infuraId}`;
};
```

Reply:
Ok, makes sense. Updated to be one function with infura id required. Everyone will need to update their env files (create one on app) for app/site to work properly. Is this intended or do we still want to fall back to defaultProvider somewhere? @Atmosfearful